e2e/loadgen/app/config.go

```diff
@@ -0,0 +1,71 @@
+package app
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/omni-network/omni/e2e/app/eoa"
+	"github.com/omni-network/omni/lib/evmchain"
+	"github.com/omni-network/omni/lib/netconf"
+)
+
+type Config struct {
+	Count        int
+	Network      string
+	Role         string
```

Comment:
I think we'll always use the XCaller role right? Not sure it has to be configurable. 

Can also use proper type otherwise.